packages/inventory/src/ducks/inventory.ts

```diff
@@ -11,18 +11,29 @@ import { Store } from 'redux';
 import { createSelector } from 'reselect';
 import { Dictionary } from '@onaio/utils';
 
+/** interface custom properties for inventory */
+export interface CustomProperties {
+  'PO Number': string;
+  'UNICEF section': string;
+}
+
 /** interface inventory */
 export interface Inventory {
```

Comment:
@HaseemUlHaq we should have retained the previous interface since it describes a valid post inventory schema.

Reply:
the previous interface describes what the post payload will look like, its going to be used in creating inventory util functions

Reply:
So [previous interface](https://github.com/OpenSRP/web/issues/405#issue-795839652) describes the types for `customProperties`?